bigtable/tests/instance_admin_integration_test.cc

```diff
@@ -47,6 +47,18 @@ class InstanceAdminIntegrationTest : public ::testing::Test {
 bool UsingCloudBigtableEmulator() {
   return std::getenv("BIGTABLE_EMULATOR_HOST") != nullptr;
 }
+bool IsInstancePresent(
+    std::vector<::google::bigtable::admin::v2::Instance> instances,
+    std::string instance_name) {
```

Comment:
clang-tidy will probably recommend passing by const reference in this case.

Reply:
fixed